packages/devtools_app/lib/src/service/service_extension_manager.dart

```diff
@@ -57,8 +58,8 @@ class ServiceExtensionManager extends Disposer {
 
   Map<IsolateRef, List<AsyncCallback>> _callbacksOnIsolateResume = {};
 
-  ConnectedApp get connectedApp => _connectedApp;
-  ConnectedApp _connectedApp;
+  ConnectedApp? get connectedApp => _connectedApp;
+  ConnectedApp? _connectedApp;
```

Comment:
this should be able to be late and non-nullable. It is initialized in vmServiceOpened

Reply:
done